build.sbt

```diff
@@ -104,7 +104,9 @@ val commonSettings =
         "-Xlint:deprecation",
         "-Xlint:unchecked"
       ),
-      assemblyMergeStrategy in assembly := nussknackerMergeStrategy
+      assemblyMergeStrategy in assembly := nussknackerMergeStrategy,
+      coverageMinimum := 60,
```

Comment:
is it enough?

Reply:
For now we have about `70%`. IMO coverage reports are "nice to have" but it shouldn't be blockers. Sometimes it producing fake data.

Reply:
true, the fact that integration tests do not count is kind of PITA...